modules/behavioural_qc/jsx/behavioural_qc_module.js

```diff
@@ -199,12 +199,8 @@ class InstrumentConflictsRow extends Component {
               <a href={baseURL + '/' + row.CandID + '/'}>{row.PSCID}</a>
           </td>
         <td>
-          <a href="#" onClick={loris.loadFilteredMenuClickHandler(
-                 'conflict_resolver/',
-                  {CandID: row.CandID,
-                  Instrument: row.TableName,
-                  Question: row.FieldName}
-          )}>{row.test_name_display}</a>
+          <a href={baseURL + '/conflict_resolver/?pSCID=' + row.PSCID +
+           '&instrument=' + row.TableName}>{row.test_name_display}</a>
```

Comment:
This seems to have lost the field name part of the filter

Reply:
could catch